### ISSUE TITLE
search filters

### ISSUE THREAD

**Issue by test2a** [stom79/TubeLab#60]
### Issue

search on web, the within instance one has many filters and sort options. 

### Steps for reproducing the issue

---

Instance:

- [ ] TubeLab
- [ ] TubeAcad

Version of TubeLab:

Android version:

**Comment by stom79** [comment]
Yes, I already use filters for sepia search. I should bring the same logic.
